tailwind.config.js

```diff
@@ -181,20 +183,51 @@ module.exports = {
           10: "#ad35bc",
           11: "#c62d69"
         },
+        event:{
+          0: "#F9CC82",
+          1: "#F9BD72",
+          2: "#F9AE62",
+          3: "#F99F52",
+          4: "#F99042",
+          5: "#FA8654"
+        },
 
         code: {
           background: "rgba(30, 30, 30, 1)",
         },
       },
       boxShadow: {
         'custom': '0 4px 8px rgba(213, 64, 0, 0.5)'
-      }
+      },
+
+      clipPath: {
+        'triangle-1': 'polygon(0 0, 100% 0, 0 100%)',
+        'triangle-2': 'polygon(100% 100%, 100% 0, 0 100%)',
+      },
     },
   },
   plugins: [
     require("@tailwindcss/forms"),
     require("@tailwindcss/typography"),
     require("@tailwindcss/aspect-ratio"),
+    function ({ addUtilities }) {
+      const newUtilities = {
+        '.clip-triangle-1': {
```

Comment:
`clip-triangle-1` class seems to be unused, since all the divs that have it are commented out